qt/settingsdialog.py

```diff
@@ -23,9 +24,40 @@
 import grp
 import re
 
-from PyQt5.QtGui import *
-from PyQt5.QtWidgets import *
-from PyQt5.QtCore import *
+from PyQt5.QtGui import QIcon, QFont, QPalette, QBrush, QColor
```

Comment:
Import * was a bad practice and should be replaced be importing the module (see https://google.github.io/styleguide/pyguide.html#22-imports).

Since this would break a lot of code that uses the imported classes I consider this change as the best work-around for now until we do a full redesign.